pkg/cri/sbserver/container_create.go

```diff
@@ -223,7 +223,7 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 		for _, v := range volumeMounts {
 			mountMap[filepath.Clean(v.HostPath)] = v.ContainerPath
 		}
-		opts = append(opts, customopts.WithVolumes(mountMap))
+		opts = append(opts, customopts.WithVolumes(mountMap, image.ImageSpec.OS))
```

Comment:
may need other fields in the ImageSpec over time if variants change or this gets more complicated.. suppose passing in just OS is fine for now

Reply:
The interface will be more consistent passing in the `Platform` object. We commonly use that type in our interfaces.

Reply:
@dmcgowan Changes made. PTAL.

Failures seem unrelated.